### PR TITLE
Support wkhtmltopdf

### PR DESCRIPTION
**Pull request by kamontat**
# Proposed Changes

> (Describe the changes and rationale behind them)

## Related Issues

> ([Github link][autolink-references] to related issues or pull requests)

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

File: bookstack/Dockerfile [hunk 1]
```diff
@@ -27,6 +27,7 @@ RUN \
         php7-tokenizer=7.3.17-r0 \
         php7-xml=7.3.17-r0 \
         php7=7.3.17-r0 \
+        wkhtmltopdf \
```
Comment (frenck) [inline bookstack/Dockerfile:30]: Please pin the version.

File: bookstack/Dockerfile [hunk 2]
```diff
@@ -47,6 +48,9 @@ RUN \
          /var/www/bookstack/tests \
          /var/www/bookstack/dev
 
+# PDF Rendering
+ENV WKHTMLTOPDF /usr/bin/wkhtmltopdf
```
Comment (frenck) [inline bookstack/Dockerfile:52]: Why is this needed, we are installing a default OS package right?

Besides, please move it higher up the Docker file so we leverage the Docker layer caching better.

**Comment by sinclairpaul** [comment]
Has this been tested?

File: bookstack/Dockerfile (hunk 2 above)
Comment (sinclairpaul) [reply bookstack/Dockerfile:52]: It's not needed in the Docker file, it would be an Env var set for FPM, as per the others.

**Comment by sinclairpaul** [comment]
Although I appreciate the PR, this is no different to what I have tested when the issue was raised.  Once used the PDF's fail to render correctly using wkhtmltopdf.

Unless you have a different experience in testing I will close this out.

**Comment by sinclairpaul** [comment]
Closing out for previous stated reasons.
